### ISSUE TITLE
[FastBert on AWS Sagemaker] GPU is peaking at 100% at inference time

### ISSUE THREAD

**Issue by azagniotov** [kaushaltrivedi/fast-bert#134]
### TL;DR
I followed a blog post that described how to glue FastBert and AWS Sagemaker.
After configuring AWS Sagemaker endpoint with a GPU-backed instance via the AWS Sagemaker UI, the GPU peaks at 100% during the inference time when using a very simple input (a single sentence containing 4-5 words). Does anyone experience the same?

### Details

I followed a blog post that described how to glue FastBert and AWS Sagemaker, I trained the default model from the blog post RoBERTa (but I also tried DistilBERT) and deployed a endpoint which is backed up by a GPU-backed instance. As per TL;DR, even with a simple input, the GPU spikes up. I tried RoBERTa and DistilBERT, but I observe the same behavior. I tried playing around with different hardware profiles of GPU-backed instances, even multiple instances, but in every try the GPU spiked up.

I am not claiming that the problem is with FastBert and maybe what I am seeing has to do with something on AWS side. But, for now I am curious - has anyone else experienced a similar behavior where GPU spiked up?

**Comment by azagniotov** [comment]
Hi @kaushaltrivedi I am really curious if you have experienced the same and are able to provide any insights.

Thank you

**Comment by adelgiudice** [comment]
I'm seeing a similar issue here and it causes OOM for some reason.
